### PR TITLE
fix(header-animation): L3-3981 Desktop Header Animation

### PR DESCRIPTION
**Pull request by adietrich3074**
**Jira ticket**

[L3-3981](https://phillipsauctions.atlassian.net/browse/L3-3981)

**Summary**

Adjusted animations to leverage cubic-bezier for smoother flyout transitions 

**Change List (describe the changes made to the files)**

- Style changes for NavigationItemTrigger and and NavigationList

**Acceptance Test (how to verify the PR)**

- Compare header flyout transitions in storybook to transitions in [protopie](https://cloud.protopie.io/p/1b0a7def5ee11e505e4808b4/r/16452f8c)

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-3981]: https://phillipsauctions.atlassian.net/browse/L3-3981?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Navigation/NavigationList/_navigationList.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @use '#scss/allPartials' as *;
 
-$opacity-transition: opacity 0.25s ease-in;
-$top-transition: top 0.25s ease-in 0.1s;
+$opacity-transition: opacity 0.2s cubic-bezier(0.65, 0, 0.35, 1);
+$top-transition: top 0.3s cubic-bezier(0.4, 0, 0.2, 1);
```
Comment (davidicus) [inline src/components/Navigation/NavigationList/_navigationList.scss:4]: Are we using these anywhere?

Comment (adietrich3074) [reply src/components/Navigation/NavigationList/_navigationList.scss:4]: If you expand the preview, they are used directly below in this file. They were added as a part of another PR so those changes don't show in my PR

**Review by scottdickerson — APPROVED** [review]
LGTM

**Comment by davidicus** [comment]
I think this could be merged. any animation tweaks should be low priority follow ups after we get everything else out the door.

**Pull request merged**